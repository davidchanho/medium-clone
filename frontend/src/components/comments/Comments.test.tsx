import React from "react";
import renderer from "react-test-renderer";
import Comments from ".";
import { testPost } from "../../tests";

describe("Comments", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Comments post={testPost} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
