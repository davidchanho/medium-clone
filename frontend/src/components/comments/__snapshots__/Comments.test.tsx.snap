// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders correctly 1`] = `
Array [
  <div>
    <svg
      fill="currentColor"
      height="1em"
      stroke="currentColor"
      strokeWidth="0"
      style={
        Object {
          "color": undefined,
        }
      }
      viewBox="0 0 24 24"
      width="1em"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M20,2H4C2.897,2,2,2.897,2,4v12c0,1.103,0.897,2,2,2h3v3.767L13.277,18H20c1.103,0,2-0.897,2-2V4C22,2.897,21.103,2,20,2z M20,16h-7.277L9,18.233V16H4V4h16V16z"
      />
    </svg>
     
    0
  </div>,
  <div>
    <div>
      Responses
    </div>
    <div />
  </div>,
]
`;
